### ISSUE TITLE
Add How and Limitations Sections to Readme

### ISSUE THREAD

**Issue by dit7ya** [Mokshit06/sveltris#2]
Thanks for the project. I am curious about how this project works and what might be the potential pitfalls. For example, I am assuming for React components to work `react` itself must be a runtime dependency, right?

**Comment by Mokshit06** [comment]
You're right, I will add that to the docs. For react components, it allows you to use either react-dom or preact for rendering. But if you're just using hooks, instead of react-dom it uses react-nil renderer as the hooks are "null" components -> they don't have any output.

**Comment by mapsgeek** [comment]
Is this possible that a project like that takes react components and `svelitize` them?
like take a react input and remove the shadow-dom stuff and make svelte compiles it as if it was just a svelte component?
imagine the react ecosystem in svelte performance <3
